crates/wysiwyg/src/dom/range.rs

```diff
@@ -357,7 +357,7 @@ mod test {
         let r = range_of(
             "\
             <ul><li>{a</li><li>b</li></ul>\
-            <ul><li>c</li><li>d</li><li>e</li></ul>fgh}|",
```

Comment:
Small explanation on this change, the join nodes mechanism is triggered by the parser, which means we do join lists of same type here, but this is not what we want to test here, so I changed to non-matching list types to fix the test (instead of recalculating the handles that are transformed by joining nodes).